app/services/send_expired_letter_notifications.rb

```diff
@@ -1,28 +0,0 @@
-class SendExpiredLetterNotifications
-  def call
-    notifications_sent = 0
-    GpoConfirmationCode.where(
-      'created_at < ?', Time.zone.now - IdentityConfig.store.usps_confirmation_max_days.days
-    ).where(bounced_at: nil, letter_expired_sent_at: nil).
-      order(created_at: :asc).each do |gpo_confirmation_code|
-      mark_sent_and_send_email(gpo_confirmation_code)
-      notifications_sent += 1
-    end
```

Comment:
Do we still need this related index and column?

https://github.com/18F/identity-idp/blob/43b72ac56125e524352314ed12a0c2fe031a8c89/db/schema.rb#L668-L669

Reply:
I am keeping my eyes on [this ticket](https://cm-jira.usa.gov/browse/LG-3721) before I drop the column and index. I link to this comment with a note that we should drop this columns if it gets cancelled.